# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/meta/websockets/bufferedAmount-unchanged-by-sync-xhr.any.js.ini
# Commit: 85883a720dab
# Full Hash: 85883a720dab61e82bcdb7391236dc06427466ee
# Author: moz-wptsync-bot <wptsync@mozilla.com>
# Date: 2021-04-07 15:47:02
# Regressor Bug: 1702830
# File Overlap Count: 1
# Description:
#   Bug 1702830 - [wpt-sync] Update web-platform-tests to 336ec39f06d15863a4af6b2aa6592530b37a5aee, a=testonly
#   
#   MANUAL PUSH: wpt sync bot
#   
#   wpt-head: 336ec39f06d15863a4af6b2aa6592530b37a5aee
# ==============================================================================

diff -r acef12d74007 -r 85883a720dab testing/web-platform/meta/websockets/bufferedAmount-unchanged-by-sync-xhr.any.js.ini
--- /dev/null	Thu Jan 01 00:00:00 1970 +0000
+++ b/testing/web-platform/meta/websockets/bufferedAmount-unchanged-by-sync-xhr.any.js.ini	Fri Apr 02 22:06:52 2021 +0000
@@ -0,0 +1,57 @@
+[bufferedAmount-unchanged-by-sync-xhr.any.sharedworker.html]
+
+[bufferedAmount-unchanged-by-sync-xhr.any.sharedworker.html?wss]
+
+[bufferedAmount-unchanged-by-sync-xhr.any.html]
+
+[bufferedAmount-unchanged-by-sync-xhr.any.html?wss]
+
+[bufferedAmount-unchanged-by-sync-xhr.any.worker.html]
+
+[bufferedAmount-unchanged-by-sync-xhr.any.sharedworker.html?wpt_flags=h2]
+  expected:
+    if (os == "linux") and webrender and not debug and fission: ["OK", "TIMEOUT"]
+    if (os == "linux") and webrender and not debug and not fission: ["OK", "ERROR", "TIMEOUT"]
+    if (os == "linux") and not webrender and not debug and (processor == "x86_64"): ["OK", "TIMEOUT", "ERROR"]
+    if (os == "win") and (processor == "x86") and not debug: ["TIMEOUT", "OK"]
+    if (os == "linux") and webrender and debug: TIMEOUT
+    if (os == "linux") and not webrender and debug: TIMEOUT
+  [bufferedAmount should not be updated during a sync XHR]
+    expected:
+      if (os == "linux") and (processor == "x86_64") and webrender: ["PASS", "FAIL"]
+      if (os == "linux") and (processor == "x86_64") and not webrender: ["PASS", "FAIL"]
+      if (os == "android") and debug and webrender: ["PASS", "FAIL"]
+      if (os == "linux") and (processor == "x86"): ["PASS", "FAIL"]
+
+
+[bufferedAmount-unchanged-by-sync-xhr.any.worker.html?wss]
+
+[bufferedAmount-unchanged-by-sync-xhr.any.html?wpt_flags=h2]
+  expected:
+    if (os == "linux") and not webrender and (processor == "x86_64") and not debug: ["OK", "TIMEOUT", "ERROR"]
+    if (os == "linux") and not webrender and (processor == "x86_64") and debug: TIMEOUT
+    if (os == "linux") and webrender and debug and not swgl: ["TIMEOUT", "CRASH"]
+    if (os == "linux") and webrender and debug and swgl: TIMEOUT
+    if (os == "linux") and webrender and not debug and not fission: ["TIMEOUT", "OK", "ERROR"]
+    if (os == "linux") and webrender and not debug and fission: ["OK", "TIMEOUT", "ERROR"]
+    if (os == "linux") and not webrender and (processor == "x86"): ["OK", "TIMEOUT"]
+    if (os == "win") and (processor == "x86") and not debug: ["TIMEOUT", "OK"]
+  [bufferedAmount should not be updated during a sync XHR]
+    expected:
+      if (os == "linux") and not fission and not webrender and (processor == "x86_64"): ["PASS", "FAIL"]
+      if (os == "linux") and fission: ["PASS", "FAIL", "TIMEOUT"]
+
+
+[bufferedAmount-unchanged-by-sync-xhr.any.worker.html?wpt_flags=h2]
+  expected:
+    if (os == "linux") and webrender and not debug and not fission: ["OK", "TIMEOUT"]
+    if (os == "linux") and webrender and not debug and fission: ["TIMEOUT", "OK", "ERROR"]
+    if (os == "linux") and not webrender and (processor == "x86_64") and not debug: ["OK", "TIMEOUT", "ERROR"]
+    if (os == "linux") and not webrender and (processor == "x86_64") and debug: TIMEOUT
+    if (os == "linux") and webrender and debug: TIMEOUT
+    if (os == "linux") and not webrender and (processor == "x86"): ["OK", "TIMEOUT"]
+    if (os == "win") and (processor == "x86") and not debug: ["TIMEOUT", "OK"]
+  [bufferedAmount should not be updated during a sync XHR]
+    expected:
+      if os == "linux": ["PASS", "TIMEOUT"]
+